interfaces/automated-testing/scd/design/README.md

```diff
@@ -1,29 +0,0 @@
-This folder contains information related to the design of the SCD components of
-the uss_qualifier automated testing suite.
-
-## Architecture
-
-Interactions between the uss_qualifier automated test driver and the system
-under test (including USSs and a DSS) are summarized in the diagram below.  The
-automated test driver attempts to inject test data, consisting of flights as
-would be requested by a user, into USSs via the
-[SCD test interface](../scd.yaml).  In addition to the responses to those
-injection requests, the automated test driver also observes the consequences of
-the injections using
-[the standard DSS and USS SCD interfaces](../../../astm-utm/Protocol).
-
-![Architecture diagram](strategic_coordination_architecture.png)
-
-## Test groups
-
-The tests performed by the automated test driver are grouped into the following
-categories:
-
-* [ASTM strategic coordination](astm-strategic-coordination)
-* [U-space](u-space)
-
-## Coverage
```

Comment:
We referenced this section in a SUSI document regarding automated testing. I imagine this is something we will be able to automatically generate from the new USS Qualifier structure in the future. Perhaps in the meantime we want to keep just a note about it if someone look for it until it is documented somewhere else in the repo? (Or move it somewhere else.)

Reply:
Sounds good -- I've now moved and adapted this file (and the two associated resources) to `monitoring/uss_qualifier/scenarios/uspace/flight_auth`.